kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -234,6 +234,7 @@ public void convert(boolean heatCoolEq, String prelude, StringBuilder semantics,
                 continue;
             }
         }
+        genSubsortAxiom(Production(Seq(), Sorts.KItem(), Seq(NonTerminal(Sorts.KConfigVar()))), syntax);
```

Comment:
Can this subsort just be added to `kast.md` instead of being injected by the kompile pipeline?

Reply:
Maybe here? https://github.com/runtimeverification/k/blob/master/k-distribution/include/kframework/builtin/kast.md#basic-k-sorts

Reply:
Hmm. I thought this was already declared and was just mistakenly getting filtered out because the code thought it was builtin syntax, but I see it actually isn't. I'll give it a try. It is not entirely clear though that we won't end up with the same problem of it getting filtered out because it's not actually part of the definition, though.